cmd/collector/app/sanitizer_v2/sanitizer.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package sanitizer_v2
+
+import (
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.uber.org/zap"
+)
+
+// SanitizeSpan sanitizes/normalizes spans. Any business logic that needs to be applied to normalize the contents of a
+// span should implement this interface.
+type SanitizeTraces func(traces ptrace.Traces) ptrace.Traces
+
+// NewStandardSanitizers are automatically applied by SpanProcessor.
+func NewStandardSanitizers(logger *zap.Logger, cache Cache) []SanitizeTraces {
+	return []SanitizeTraces{
+		NewEmptyServiceNameSanitizer(),
+		NewUTF8Sanitizer(logger),
+		serviceNameSanitizer{cache: cache}.Sanitize,
```

Comment:
this was not present in v1 standard sanitizers, don't add it here

Reply:
Yes, but when we removed the newServicenameSanitizer from service_name_sanitizer.go, we are not using the functions defined in it and throwing error of not used function 😅